datafusion/sql/src/unparser/expr.rs

```diff
@@ -261,26 +261,57 @@ impl Unparser<'_> {
                     uses_odbc_syntax: false,
                 }))
             }
-            Expr::SimilarTo(Like {
+            Expr::Like(Like {
```

Comment:
Why did you reverse the order the `Like` and `SimilarTo` were handled?

Reply:
My mistake, fixed